test/fixtures/changes.yml

```diff
@@ -0,0 +1,15 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
```

Comment:
If these files exist, why not use them in tests?

Reply:
this were created automatically when i generated the project and i forgot to remove them